docs/reference/import.md

```diff
@@ -28,8 +29,58 @@ Unfortunately, copying data is not always feasible for the following reasons:
 2. It requires you to stop making changes to the data before starting to copy.
 3. It requires you to switch to using the lakeFS endpoint in all places at once.
 
-## Using lakeFS import tool
-To solve this we offer an import tool that does not copy any data, allowing for a more gradual onboarding process.
+## Importing data from an object store without actually copying it
+
+The `lakectl` command supports ingesting objects from a source object store without actually copying the data itself.
+This is done by listing the source bucket (and optional prefix), and creating pointers to the returned objects in lakeFS.
+
+By doing this, it's possible to take even large sets of objects, and have them appear as objects in a lakeFS branch, as if they were written directly to it.
+
+For this to work, we'd need to ensure 2 things first:
+
+1. The user calling `lakectl ingest` must have permissions to list the object at the source object store
+1. The lakeFS installation must have read permissions to the objects being ingested
+
+### Running lakectl ingest with S3 as the source
+
+```shell
+lakectl ingest \
+  --from s3://bucket/optional/prefix/ \
+  --to lakefs://my-repo/ingest-branch/optional/path/
+```
+
+The `lakectl ingest` command will attempt to use the current user's existing credentials and will respect instance profiles, 
+environment variables and credential files [in the same way that the AWS cli does](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html){: target="_blank" }
+
+### Running lakectl ingest with Azure Blob storage as the source
+
+```shell
+export AZURE_STORAGE_ACCOUNT="storageAccountName"
+export AZURE_STORAGE_ACCESS_KEY="EXAMPLEroozoo2gaec9fooTieWah6Oshai5Sheofievohthapob0aidee5Shaekahw7loo1aishoonuuquahr3=="
+lakectl ingest \
+   --from https://storageAccountName.blob.core.windows.net/container/optional/prefix/ \
+   --to lakefs://my-repo/ingest-branch/optional/path/
+```
+
+The `lakectl ingest` command currently supports storage accounts configured through environment variables as shown above.
+
```

Comment:
We might want to note that we only support `http` and `https` (not `wasb` `abfs` `adls`)

Reply:
Thanks, added a note about that.